.cirrus.default.yml

```diff
@@ -149,7 +149,7 @@ validate_task:
   eks_container:
     <<: *BUILDER_CONTAINER_DEFINITION
     cpu: 4
-    memory: 8G
```

Comment:
Just to double check - reducing the spec is on purpose?

Reply:
Yes, I've checked the statistics and DevInfra asked to lower the specs if they're not needed. At most I used 2GB and sometimes it spikes to 3GB and therefore 4GB is enough.